### DISCUSSION TITLE
Commit as repo owner when logged in different account on git desktop?

### DISCUSSION THREAD

**Discussion by R3uan3** [community/community#34101] [Other Feature Feedback, Questions, & Ideas] [answered]
When I'm logged to `GitHub Desktop` with account `XXX`, and commit a file using a token to a repo of a different account `YYY`:
```
git add .
git commit -m "..."
git push https://<USER>:<TOKEN>@github.com/.../... main
```

The file is uploaded to `YYY` repo and on Github website it displays the files has been committed by the `XXX` account.

How do i upload it as if it where being uploaded from `YYY` account and not `XXX`?

Do i need to logout from `Github Desktop` first 😢?

**Comment by airtower-luna** [comment, marked as answer]
Which account a commit is assigned to has absolutely *nothing* to do with the token used to push or how you're logged in. GitHub uses the author email (and committer email, which may be different) recorded in the commit to detect which user created a commit.

Usually you [set the author email and name in your Git config](https://git-scm.com/docs/git-config#Documentation/git-config.txt-username) (`user.email` and `user.name`), if you want to override it for a specific commit you can use [`git commit --author=...`](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---authorltauthorgt).
